composer.json

```
@@ -26,19 +26,18 @@
         "forum": "https://community.contao.org"
     },
     "require": {
-        "php": "^7.1 || ^8.0",
-        "contao/core-bundle": "^4.9",
-        "codefog/contao-haste": "^4.13",
-        "menatwork/contao-multicolumnwizard-bundle": "^3.3"
+        "php": "^7.4 || ^8.0",
+        "contao/core-bundle": "^4.9 || ^5.0",
```

```suggestion
        "contao/core-bundle": "^4.13 || ^5.0",
```

Haste 5 is not compatible with < 4.13 so no need to let Composer load the package information of versions not resolvable anyway.